src/runner/index.js

```diff
@@ -33,6 +35,13 @@ export default class Runner {
     // Run task
     _runTask (Reporter, browserConnections, tests) {
         return new Promise((resolve, reject) => {
+            if (browserConnections.some(connection => connection.disconnected)) {
```

Comment:
AFAIK we don't need this check here, because `_runTask` call happens synchronously right after the browser bootstrapping